### DISCUSSION TITLE
How to  cancel keyword italics

### DISCUSSION THREAD

**Discussion by JianwenShaw** [LazyVim/LazyVim#3681] [Q&A] [answered]
I'm new to use LazyVim,so I haven't change any cofigurations.Everything is as it was when I first downloaded it.Now I have trouble with how to make the keyword regular instead of italics?
![image](https://github.com/LazyVim/LazyVim/assets/118002258/1f360515-8a20-403d-ac4e-b3f1949bfef8)

**Comment by hirotaka** [comment, marked as answer]
I also prefer non-italic, so I add the following to `plungins/ui.lua`, for example. The file name could be anything.

```lua
return {
  "folke/tokyonight.nvim",
  opts = {
    styles = {
      keywords = { italic = false },
      comments = { italic = false },
    },
  },
}
```

**Reply by JianwenShaw** [reply]
Wow!I did it.Your reply had a good effect.Thank you for your enthusiasm
